src/pages/policy/input/ParameterEditor.jsx

```diff
@@ -30,6 +31,17 @@ export default function ParameterEditor(props) {
     startDate,
   );
 
+  useEffect(() => {
+    const [parameterStartDate, parameterEndDate] = policy.reform.data[
```

Comment:
```suggestion
    const [parameterStartDate, parameterEndDate] = policy?.reform?.data[
```

Reply:
This is to avoid runtime errors if, for example, `policy.reform` exists, but the `data` key does not